Gemfile

```diff
@@ -48,6 +54,10 @@ group :development, :test do
   gem 'rspec-rails'
   gem 'rubocop-rspec'
   gem 'selenium-webdriver'
+  # Factory bot to add fixtures to the tests
+  gem 'factory_bot_rails', '~> 4.0'
+  # Use faker to fill random data for tests
+  gem 'faker', git: 'https://github.com/stympy/faker.git', branch: 'master'
```

Comment:
@sinpai can't we fetch `faker` gem without specifying `git` and `branch` options? Is there any issue with it?

Reply:
It's from official github page (advised to use like this)